### DISCUSSION TITLE
Dropdown and other elevated elements are hard to see in dark themes

### DISCUSSION THREAD

**Discussion by ffxsam** [saadeghi/daisyui#2038] [Ideas / feature requests]
In light themes, it's easy enough to see elevation/separation from the page for a dropdown, due to the shadow:

<img width="237" alt="CleanShot 2023-06-21 at 20 09 56" src="https://github.com/saadeghi/daisyui/assets/12532733/f6f2c89d-4edd-4ebd-bbbe-471807cb58a9">

But for dark themes, the dropdown blends into the page and is difficult to discern:

<img width="235" alt="CleanShot 2023-06-21 at 20 10 24" src="https://github.com/saadeghi/daisyui/assets/12532733/b92001eb-957d-4c6c-b398-ff0c897d73e2">

Google Material Design solves this by using a slightly lighter shade for elevated items (only in dark mode). It could look like this, for example:

<img width="236" alt="CleanShot 2023-06-21 at 20 13 32" src="https://github.com/saadeghi/daisyui/assets/12532733/41bc3305-adaa-4897-8ff0-be3e4ace6c45">

I wanted to start a discussion here before opening an issue.

**Comment by saadeghi** [comment]
It's a problem with the examples on docs not the component style itself.  
I used `bg-base-100` background color which is the same as the background of the preview box. I will fix this soon.
